webrtc.html

```diff
@@ -3998,6 +3998,19 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             the <a><code>RTCRtpReceiver</code></a> to be stopped.</p>
           </dd>
 
+          <dt>void activateSender()</dt>
+
+          <dd>
+            <p>The <dfn id="dom-rtptransceiver-activatesender"><code>activateSender</code></dfn>
+            method activates the <code>RTCRtpTransceiver.sender</code> such that future calls
```

Comment:
we're moving away from using Foo.bar to refer to attribute (see #316); could use just use `<code>sender</code>`?